src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -23,6 +23,27 @@ namespace Xamarin.Android.Build.Tests
 	[Parallelizable (ParallelScope.Children)]
 	public partial class BuildTest : BaseTest
 	{
+		[Test]
+		[Category ("SmokeTests")]
+		public void SmokeTestBuildWithSpecialCharacters ([Values (false, true)] bool forms)
+		{
+			var testName = "テスト";
+
+			var rootPath = Path.Combine (Root, "temp", TestName);
+			var proj = forms ?
+				new XamarinFormsAndroidApplicationProject () :
+				new XamarinAndroidApplicationProject ();
```

Comment:
The problem, as outlined [earlier](https://github.com/xamarin/xamarin-android/pull/6273#issuecomment-915683031), is that the MSBuildDeviceIntegration tests only run on macOS, which doesn't have a "non-ASCII characters" problem in the first place.

What we want/need is for MSBuildDevice integration to run on *Windows*.  The problem is we tried that in an earlier iteration of https://github.com/xamarin/xamarin-android/pull/6297, and hit https://github.com/xamarin/xamarin-android/pull/6297#issuecomment-920309384

> Instead, it (later) insta-fails, because of virtualization:
> 
>      [emulator stdout] emulator: ERROR: x86_64 emulation currently requires hardware acceleration! (TaskId:5)
>      C:\a\_work\1\s\build-tools\scripts\TestApks.targets(65,5): error : Emulator failed to start: emulator: ERROR: x86_64 emulation currently requires hardware acceleration! [C:\a\_work\1\s\tests\Mono.Android-Tests\Mono.Android-Tests.csproj]
>      [emulator stdout] CPU acceleration status: Hyper-V detected and Windows Hypervisor Platform is available. Please ensure both the "Hyper-V" and "Windows Hypervisor Platform" features enabled in "Turn Windows features on or off". (TaskId:5)
>      [emulator stdout] More info on configuring VM acceleration on Windows: (TaskId:5)
>      [emulator stdout] https://developer.android.com/studio/run/emulator-acceleration#vm-windows (TaskId:5)
>      [emulator stdout] General information on acceleration: https://developer.android.com/studio/run/emulator-acceleration. (TaskId:5)

@pjcollins had additional followup: https://github.com/xamarin/xamarin-android/pull/6297#issuecomment-920334650

> * Investigate Helix machine pools for Android emulator testing.
> * Revisit physical Windows machine usage. One possible target would be the shared pool we used to run QA tests on. Some of these machines have physical Android devices attached, but could still potentially be used: …
> * See if we can somehow enable virtualization on our 1ES Windows pool.

*In the meantime*, we're going to need to rely on manual testing for Windows (ugh).